uwp/Hub-Tile/SfpulsingTile.md

```diff
@@ -116,7 +116,7 @@ Set the image as Content of `SfPulsingTile` as given below:
 
 {% endtabs %}
 
-![Pulsing tile displayed bitmap](SfpulsingTile-images/SfpulsingTile-img1.jpeg)
+![UWP SFPulsing tile displayed bitmap](sfpulsingtile-images/uwp-hub-tile-adding-image-to-sfpulsingtile.jpeg)
```

Comment:
double time mentioned control name in image name. 
Pulsing tile is the control name here. So, change like as below.
adding-images-in-uwp-pulsing-tile

Reply:
resolved